src/vorta/views/repo_tab.py

```diff
@@ -198,8 +198,7 @@ def create_ssh_key(self):
         ssh_add_window = SSHAddWindow()
         self._window = ssh_add_window  # For tests
         ssh_add_window.setParent(self, QtCore.Qt.WindowType.Sheet)
-        ssh_add_window.accepted.connect(self.init_ssh)
-        # ssh_add_window.rejected.connect(lambda: self.sshComboBox.setCurrentIndex(0))
+        ssh_add_window.rejected.connect(self.init_ssh)
```

Comment:
This fixes a bug I found during testing. Before, the SSH Key selector was not repopulating with new keys until Vorta was restarted. Now, it will repopulate whenever the user closes `ssh_add_window`